### ISSUE TITLE
 Amazon SES / MailGun issues as SMTP with FusionAuth 

### ISSUE THREAD

**Issue by jdegger** [FusionAuth/fusionauth-issues#1263]
### Description 
On a deployment setup by FusionAuth, I'm not able to send e-mails with valid mailgun credentials. Somebody else reported the same thing for SES.

### Affects versions
1.28.0

### Steps to reproduce
1. enter mailgun smtp
2. test e-mail
3. failure

### Expected behavior
Send a e-mail succesfully

### Logs
```
Unable to send email via JavaMail
Prime Messaging Exception
```

Enabling SMTP logs in advanced settings did not do anything, nothing is visible in the FusionAuth logs.

### Platform
FusionAuth deployed service instance

### Additional context
https://fusionauth.io/community/forum/topic/1075/amazon-ses-issues-as-smtp-with-fusionauth

**Comment by jobannon** [comment]
Hi @jdegger,

I can test this locally.  Depending on the configuration of the mail service you may need to configure additional parameters than are included by default in the JavaMail, such as TLS version required.

You can get additional error information by enabling debug under `tenant` -> `advanced` -> `SMTP settings`

`mail.debug=true`

More information can be found in our troubleshooting guide.  
https://fusionauth.io/docs/v1/tech/troubleshooting/#troubleshooting-email

I will post back any other findings in the meantime.

Thanks for the report!


Thanks,
Josh

**Comment by jdegger** [comment]
Hi @jobannon, thanks for your reply. We already tried mail.debug=true but nothing was shown in the logs.

I figured out the issue though: outgoing connections on port 25 are blocked on FusionAuth servers deployed on company.fusionauth.io.

It might be a good idea for FusionAuth to hint at connection timeouts and firewall settings on their servers, as it is managed by them.

---------

Closing this issue, for anybody encountering the same problem: 
1. Try switching to another port (587, 465, or 2525)
2. Try switching TLS to SSL and the other way around with all the different port configurations.
Took me a while to find this, probably because I mistyped 587 as 578.
